docs/REST API/rest-api.md

```diff
@@ -3051,6 +3051,7 @@ Returns the information of the specified host.
           "name": "profile1",
           "status": "verifying",
           "operation_type": "install",
+          "scioe": "device",
```

Comment:
```suggestion
          "scope": "device",
          "users": [],
```

I would recommend we have the users returned here to allow for the possible expansion of multi-user support.
empty for device and no user, most likely we will see a list of 1 here but it would be possible in the future to have it applied to more than 1 user.